storage/src/main/java/com/tencent/rss/storage/handler/impl/HdfsClientReadHandler.java

```diff
@@ -51,7 +66,96 @@ public HdfsClientReadHandler(
         ShuffleStorageUtils.getShuffleDataPathWithRange(appId,
             shuffleId, partitionId, partitionNumPerRange, partitionNum));
     init(fullShufflePath);
-    readAllIndexSegments();
+    cursor = indexReaderMap
+        .entrySet()
+        .stream()
+        .sorted(Comparator.comparing(Entry::getKey))
```

Comment:
why we need sort?

Reply:
Read file by time sequnce.

Reply:
What's advantage?

Reply:
1. read earlier write early
2. read the group of files written by the same server

Reply:
`ls` will return dictionary-order file list, is it enough?

Reply:
It may not be appropriate to rely on hadoop list status api's behavior nor the map's iterator sequnce (and what's more it's hash map).

Reply:
OK